ambry-tools/src/main/java/com.github.ambry.store/HardDeleteVerifier.java

```diff
@@ -11,7 +11,7 @@
  * distributed under the License is distributed on an "AS IS" BASIS,
```

Comment:
@nsivabalan It seems like you already changed the file name to match the class name, right?